### DISCUSSION TITLE
Thanks for Dockge!

### DISCUSSION THREAD

**Discussion by hugalafutro** [louislam/dockge#534] [General]
Buddy get out of my head🤣 How do you make an app that does everything I wanted from a docker gui?

I tried portainer, rancher or w/e it was, lazydocker and I like aspects of them and still use portainer to delete unused images in bulk and lazydocker to check on logs and enter shell (well I won't anymore innit).

Either the apps can't do it or I'm too stupid to figure it out, but all I ever wanted to do from a web gui is
- to retain existing file structure  so that if the tool breaks I can still manage stacks manually and they never moved from where they were
- 1 button to update the whole stack without having to ssh into host (I didn't get the stacks management in portainer at all it didn't seem I can retain my existing stack.)
- shell / logs quickly accessible and immediately displayed without ssh (lazydocker sort of does, but if no logs for long time displays nothing and I still have to ssh, portainer is clunky)

I've no idea where I heard about dockge think it was some truenas jailmaker tutorial on youtube, I been using your uptime-kuma for years, but never checked your other stuff.

I'm instantly in love, and still a bit in shock as it isn't often you find something that does everything you were looking for and get it up and running in seconds without any hassle or compromise.

Thank you!

**Comment by Morfeosh** [comment]
Is the best for docker!!
